src/assets/config.json

```diff
@@ -22,6 +22,11 @@
   "orderTransportDurationDays": 2,
   "orderTransportCost": 50,
 
+  "increaseTruckSizeQuantity": 500,
+  "decreaseTruckSizeQuantity": 500,
```

Comment:
Are we sure we want to also offer a decrease? Similarly with the warehouse, where we do not offer such functionality: what would be the benefit?